src/test/java/org/prebid/server/auction/ExchangeServiceTest.java

```diff
@@ -387,8 +390,10 @@ public void shouldPassRequestWithInjectedSchainInSourceExt() {
         givenBidder(bidder2Name, bidder2, givenEmptySeatBid());
         givenBidder(bidder3Name, bidder3, givenEmptySeatBid());
 
-        final ObjectNode schainObject = mapper.createObjectNode().put("var", "1");
-        final ObjectNode allSchainObject = mapper.createObjectNode().put("any", "any");
+        final ObjectNode objectNode = mapper.createObjectNode().put("any", "any");
+        final ExtRequestPrebidSchainSchainNode nodes = ExtRequestPrebidSchainSchainNode.of("asi", "sid", 1, "rid", "name", "domain", objectNode);
+        final ExtRequestPrebidSchainSchain schainObject = ExtRequestPrebidSchainSchain.of("ver", 1, singletonList(nodes), objectNode);
+        final ExtRequestPrebidSchainSchain allSchainObject = ExtRequestPrebidSchainSchain.of("ver", 1, singletonList(nodes), objectNode);
```

Comment:
Please. Prepare fixture like this
```
        final ObjectNode schainExtObjectNode = mapper.createObjectNode().put("any", "any");
        final ExtRequestPrebidSchainSchainNode specificNodes = ExtRequestPrebidSchainSchainNode.of("asi", "sid", 1, "rid", "name", "domain", schainExtObjectNode);
        final ExtRequestPrebidSchainSchain specificSchain = ExtRequestPrebidSchainSchain.of("ver", 1, singletonList(specificNodes), schainExtObjectNode);
        final ExtRequestPrebidSchain schainForBidders = ExtRequestPrebidSchain.of(Arrays.asList(bidder1Name, bidder2Name), specificSchain);

        final ExtRequestPrebidSchainSchainNode generalNodes = ExtRequestPrebidSchainSchainNode.of("t", null, 0, "a", null, "ads", null);
        final ExtRequestPrebidSchainSchain generalSchain = ExtRequestPrebidSchainSchain.of("t", 123, singletonList(generalNodes), null);
        final ExtRequestPrebidSchain allSchain = ExtRequestPrebidSchain.of(singletonList("*"), generalSchain);
``` 
To test real behavior. 

You need to verify that there some changes for specified bidders, and there are different changes for rest of them.